Cabal/Cabal.sublime-syntax

```diff
@@ -0,0 +1,91 @@
+%YAML 1.2
+---
+# Derived from https://github.com/JustusAdam/language-haskell/blob/2859b32113407977f0edcb6c19f0fe3eaae1294f/syntaxes/cabal.tmLanguage
+name: Cabal
+file_extensions:
+  - cabal
+scope: source.cabal
+contexts:
+  main:
+    - match: |-
+        (?ix)(\n|^)
+            ( name
+            | version
+            | cabal-version
+            | build-type
+            | license(-file)?
+            | copyright
+            | author
+            | maintainer
+            | stability
+            | homepage
+            | bug-reports
+            | package-url
+            | synopsis
+            | data-(files|dir)
+            | description
+            | category
+            | extra-(source|doc|tmp)-files
+            | tested-with
+            ):
```

Comment:
`:` is punctuation, should receive a punctuation scope and be exempted from the keyword scope.